test/e2e/wdio.conf.js

```diff
@@ -54,7 +54,7 @@ exports.config = {
     ],
     // Patterns to exclude.
     exclude: [
-        // 'path/to/excluded/files'
+        './specs/**/refreshTokens*.js'
```

Comment:
Regular e2e test suite will not run e2e tests, as refresh token feature is NOT enabled by default for all orgs.